site/docs/aws.md

```diff
@@ -22,24 +22,34 @@ This section describes how to use Iceberg with AWS.
 
 ## Enabling AWS Integration
 
-The `iceberg-aws` module is bundled with Spark and Flink engine runtimes.
+The `iceberg-aws` module is bundled with Spark and Flink engine runtimes for all versions after `0.11.0`.
 However, the AWS clients are not bundled so that you can use the same client version as your application.
 You will need to provide the AWS v2 SDK because that is what Iceberg depends on.
 You can choose to use the [AWS SDK bundle](https://mvnrepository.com/artifact/software.amazon.awssdk/bundle), 
 or individual AWS client packages (Glue, S3, DynamoDB, KMS, STS) if you would like to have a minimal dependency footprint.
 
+All the default AWS clients use the [URL Connection HTTP Client](https://mvnrepository.com/artifact/software.amazon.awssdk/url-connection-client/2.15.61)
```

Comment:
Nit: might be good to directly link to https://mvnrepository.com/artifact/software.amazon.awssdk/url-connection-client (without linking into any specific version)?

Reply:
Nice catch, I accidentally included the version